regress/loader/TestSkipANALYZE-pre.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA pgreg;
```

Comment:
Just to reduce changes, how about loading the shapefile in the default schema rather than forcing `pgreg` ? It would let you drop both the pre and the post sql scripts

Reply:
Yes, I'm definitely for simplification whenever possible. Now the test table is in the default schema (public).